python/dolfinx/la.py

```diff
@@ -245,6 +250,18 @@ def array(self) -> np.ndarray:
         """Local representation of the vector."""
         return self._cpp_object.array
 
+    @property
+    def vector(self):
```

Comment:
I think we need a more descriptive name. Maybe `petsc_vec`?

Reply:
I also prefer `petsc_vec`. I originally used `vector` since this was the method used in the `fem.Function` class.
I will make the update.

Reply:
@garth-wells : This has been renamed as recommended.